src/Routes/Systems/Systems.js

```diff
@@ -22,6 +23,8 @@ const Systems = () => {
     const intl = useIntl();
     const sigPageData = useQuery(GET_SIGNATURE_PAGE);
     const chartCmpData = useQuery(GET_TIME_SERIES_STATS);
+    const chrome = useChrome();
+    chrome.updateDocumentTitle(`Systems - Malware | Red Hat Insights`);
```

Comment:
These should maybe be wrapped in a `useEffect` and have `chrome` as a dependency.

Reply:
fair enough, this makes sense. I'll update it now